<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntry</class>
 <widget class="QWidget" name="DataEntry">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>692</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>692</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>692</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>44</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>44</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>440</width>
     <height>44</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px; 
} </string>
   </property>
   <property name="text">
    <string>PZB列车数据输入</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_1_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>44</y>
     <width>294</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
background: rgb(86, 86, 86); 
} </string>
   </property>
   <property name="text">
    <string>BRA  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_2_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>116</y>
     <width>294</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
background: rgb(86, 86, 86); 
} </string>
   </property>
   <property name="text">
    <string>BRH  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_3_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>188</y>
     <width>294</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
background: rgb(86, 86, 86); 
} </string>
   </property>
   <property name="text">
    <string>Train length  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_4_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>294</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
background: rgb(86, 86, 86); 
} </string>
   </property>
   <property name="text">
    <string>VMZ  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fielddata_1_lineEdit">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>44</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
font-size:15pt;
color:white;
border: 0px;
background: rgb(85, 85, 85); 
}  
QLineEdit:focus {
font-size:15pt;
border:0px;
color:black;
background: rgb(150, 150, 150); 
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fielddata_2_lineEdit">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>116</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
font-size:15pt;
color:white;
border: 0px;
background: rgb(85, 85, 85); 
}  
QLineEdit:focus {
font-size:15pt;
border:0px;
color:black;
background: rgb(150, 150, 150); 
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fielddata_3_lineEdit">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>188</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
font-size:15pt;
color:white;
border: 0px;
background: rgb(85, 85, 85); 
}  
QLineEdit:focus {
font-size:15pt;
border:0px;
color:black;
background: rgb(150, 150, 150); 
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fielddata_4_lineEdit">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>260</y>
     <width>146</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
font-size:15pt;
color:white;
border: 0px;
background: rgb(85, 85, 85); 
}  
QLineEdit:focus {
font-size:15pt;
border:0px;
color:black;
background: rgb(150, 150, 150); 
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_5_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>402</y>
     <width>294</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>BRA  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_1_label_3">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>402</y>
     <width>146</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt; 
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_6_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>438</y>
     <width>294</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>BRH  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_1_label_5">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>438</y>
     <width>146</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt; 
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_7_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>474</y>
     <width>294</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>Train length  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_1_label_7">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>474</y>
     <width>146</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt; 
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLabel" name="field_8_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>294</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>294</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt;
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>VMZ  </string>
   </property>
  </widget>
  <widget class="QLabel" name="field_1_label_9">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>510</y>
     <width>146</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>146</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:15pt; 
color:gray;
border: 0px;
} </string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>620</y>
     <width>119</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>119</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>119</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>620</y>
     <width>238</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>238</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>238</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
border-top: 1px solid black;
border-left:1px solid black;  
border-bottom: 1px solid rgb(8,24,57);
border-right:1px solid rgb(8,24,57);  
} </string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btable">
   <property name="geometry">
    <rect>
     <x>357</x>
     <y>620</y>
     <width>82</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>82</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn_2">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>620</y>
     <width>238</width>
     <height>72</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>238</width>
     <height>72</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>238</width>
     <height>72</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background: rgb(6,17,35);
font-size:15pt;
color:gray;
border-left:1px solid balck;border-top:1px solid black;border-right:1px solid rgb(8,24,57);border-bottom:1px solid rgb(8,24,57);
outline: none;
} </string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <zorder>fielddata_3_lineEdit</zorder>
  <zorder>Title_label</zorder>
  <zorder>field_1_label</zorder>
  <zorder>field_2_label</zorder>
  <zorder>field_3_label</zorder>
  <zorder>field_4_label</zorder>
  <zorder>fielddata_1_lineEdit</zorder>
  <zorder>fielddata_2_lineEdit</zorder>
  <zorder>fielddata_4_lineEdit</zorder>
  <zorder>field_5_label</zorder>
  <zorder>field_1_label_3</zorder>
  <zorder>field_6_label</zorder>
  <zorder>field_1_label_5</zorder>
  <zorder>field_7_label</zorder>
  <zorder>field_1_label_7</zorder>
  <zorder>field_8_label</zorder>
  <zorder>field_1_label_9</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>okBtn</zorder>
  <zorder>btable</zorder>
  <zorder>okBtn_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
